config/converter.yml.example

```diff
@@ -1,20 +1,31 @@
-development:
-  :url:
+shared:
+  :ext:
+    - '.aif'
+    - '.brml'
+    - '.cif'
+    - '.dpt'
+    - '.dsp'
+    - '.dta'
+    - '.gz'
+    - '.pssession'
+    - '.tar'
+    - '.txt'
+    - '.xls'
+    - '.xlsx'
+    - '.xy'
   :profile:
   :secret_key:
   :timeout:
-  :ext:
+  :url:
+
+development:
+  :profile: 'chemotion'
```

Comment:
Only the profile 'dev' can be used without username and password. Hence one should use the 'dev' profile for development